### ISSUE TITLE
[Accessibility] Product single add to cart button misaligned on Zoom with longer text

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#59793]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Some extensions hook into the product single add-to cart or buy now button on the product single and change the label. For example, the Pre-Orders Extension changes the label to "Pre-order now". 

When longer text is present on this button, and the website is zoomed to 200%, words can wrap to multiple lines because there is not enough space on the screen for the button next to the quantity field in the column to the right of the product image. This makes it difficult to read the button name.

<img width="1702" height="725" alt="Pre-order now button on the post single with order broken to two lines" src="https://github.com/user-attachments/assets/aee9e5cb-a988-4302-b033-db2f56ae6953" />

### Expected behavior

Words should not break to multiple lines on zoom.

Suggested fix is to have the quantity field stack above the button in this scenario.

You may also want to have reduce the flex-basis on the column containing the image from 512px to 412px or set a max width of 50% on that column to give more space for text rather than the image. On zoom, the image is commanding a lot of the screen and taking space away from the product information and the add-to-cart tools. 

### Actual behavior

The button text wraps to two lines.

### Steps to reproduce

You can view this issue at [https://woo.equalizedigital.com/product/pre-order-product/](https://woo.equalizedigital.com/product/pre-order-product/) (password: wooa11y)

With the pre-orders extension active and in an FSE theme or block-based product page:

1. Go to a pre-order product single.
2. Zoom your browser to 200%.

You can also fake this by inspecting the button on any product page and changing the label text to "Pre-order now" and then zooming to 200%.

### WordPress Environment

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 10.1.0-dev
Legacy REST API Package Version: The Legacy REST API plugin is not installed on this site.
Action Scheduler Version: ✔ 3.9.2
Log Directory Writable: ✔
WP Version: 6.8.2
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
Server Architecture: Linux 4.19.286-203.ELK.el7.x86_64 x86_64
PHP Version: 7.4.33
PHP Post Max Size: 516 MB
PHP Time Limit: 60
PHP Max Input Vars: 1000
cURL Version: 8.7.1
OpenSSL/1.1.1w

SUHOSIN Installed: –
MySQL Version: 5.7.23-23
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

attachment: 59
page: 11
post: 2
product: 58
product_variation: 160
revision: 43
shop_coupon: 5
shop_order_placehold: 69
wp_global_styles: 2
wp_navigation: 1
wp_template: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (20) ###

The Bluehost Plugin: by Bluehost – 4.4.1
Disable Emails: by WebAware – 1.8.2
Password Protected: by Password Protected – 2.7.9
PU2 DevTools: by  –
User Switching: by John Blackbourn – 1.9.2
WooCommerce Smooth Generator: by Automattic – 1.2.0
WooCommerce Back In Stock Notifications: by Woo – 3.3.101-dev
WooCommerce Composite Products: by Woo – 10.7.100-dev
WooCommerce Conditional Shipping and Payments: by Woo – 2.2.0
WooCommerce Payfast Gateway: by WooCommerce – 1.7.1
Braintree for WooCommerce Payment Gateway: by WooCommerce – 3.2.9
WooCommerce Gift Cards: by Woo – 2.5.0
WooCommerce Min/Max Quantities: by Woo – 5.2.5
WooPayments: by WooCommerce – 9.6.0
WooCommerce Pre-Orders: by WooCommerce – 2.2.8
WooCommerce Product Add-Ons: by Woo – 7.9.0
WooCommerce Product Bundles: by Woo – 8.4.0
WooCommerce Subscriptions: by WooCommerce – 7.6.0
WooCommerce: by Automattic – 10.1.0-dev
WP Mail Logging: by WP Mail Logging Team – 1.14.0

### Inactive Plugins (5) ###

Akismet Anti-spam: Spam Protection: by Automattic - Anti-spam Team – 5.5
Jetpack: by Automattic – 14.8
UpdraftPlus - Backup/Restore: by TeamUpdraft
DavidAnderson – 1.25.6

WooCommerce Product Recommendations: by Woo – 4.1.5
WooCommerce Stripe Gateway: by Stripe – 9.6.0

### Must Use Plugins (1) ###

SSO: by Garth Mortensen
Mike Hansen – 0.5


### Settings ###

Legacy API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: bundle (bundle)
composite (composite)
external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: ✔
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: –
Enabled Features: analytics
marketplace
order_attribution
site_visibility_badge
remote_logging
email_improvements
blueprint
point_of_sale
custom_order_tables


### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 14 MB

### WC Pages ###

Shop base: #6 - /shop/
Cart: #7 - /cart/ -  Contains the woocommerce/cart block
Checkout: #8 - /checkout/ -  Contains the woocommerce/checkout block
My account: #9 - /my-account/ -  Contains the [woocommerce_my_account] shortcode
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-Five
Version: 1.2
Author URL: https://wordpress.org
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
Theme type: Block theme
WooCommerce Support: ❌ Not declared

### Templates ###

Overrides: /home2/amberhin/public_html/website_f5ae36ad/wp-content/plugins/woocommerce/templates/block-notices/error.php
/home2/amberhin/public_html/website_f5ae36ad/wp-content/plugins/woocommerce/templates/block-notices/notice.php
/home2/amberhin/public_html/website_f5ae36ad/wp-content/plugins/woocommerce/templates/block-notices/success.php


### WooPayments ###

Version: 9.6.0
Connected to WPCOM: ❌ No
Logging: Disabled

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://woo.equalizedigital.com
Subscriptions-core Library Version: 8.3.0
Subscription Statuses: wc-active: 1
wc-cancelled: 1
wc-on-hold: 5
wc-pending-cancel: 1

WooCommerce Account Connected: ✔ Yes
Active Product Key: ❌ No

### Store Setup ###

Country / State: United States (US) — Texas

### Subscriptions by Payment Gateway ###

Direct bank transfer: wc-cancelled: 1
wc-on-hold: 4

other: wc-active: 1
wc-pending-cancel: 1


### Payment Gateway Support ###

Direct bank transfer: products
Check payments: products
Cash on delivery: products
Braintree (Credit Card): products
card_types
payment_form
tokenization
charge
charge-virtual
authorization
capture_charge
customer_decline_messages
refunds
voids
customer_id
add_payment_method
token_editor
apple_pay
subscriptions
subscription_suspension
subscription_cancellation
subscription_reactivation
subscription_amount_changes
subscription_date_changes
multiple_subscriptions
subscription_payment_method_change_customer
subscription_payment_method_change_admin
pre-orders

Braintree (PayPal): products
card_types
payment_form
tokenization
charge
charge-virtual
authorization
capture_charge
customer_decline_messages
refunds
voids
customer_id
add_payment_method
token_editor
subscriptions
subscription_suspension
subscription_cancellation
subscription_reactivation
subscription_amount_changes
subscription_date_changes
multiple_subscriptions
subscription_payment_method_change_customer
subscription_payment_method_change_admin
pre-orders


### Back In Stock ###

Database Version: 3.3.101
Loopback Test: ✔

### Composite Products ###

Database Version: 10.7.100
Loopback Test: ✔
Template Overrides: –

### Gift Cards ###

Database Version: 2.5.0
Loopback Test: ✔
Task Queueing Test: ✔

### Product Bundles ###

Database Version: 8.4.0
Loopback Test: ✔
Template Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
onboarding
onboarding-tasks
pattern-toolkit-full-composability
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
printful
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store
add-to-cart-with-options-stepper-layout

Disabled Features: product-data-views
experimental-blocks
experimental-iapi-mini-cart
experimental-iapi-runtime
coming-soon-newsletter-template
minified-js
product-pre-publish-modal
settings
async-product-editor-category-field
product-editor-template-system
use-wp-horizon
reactify-classic-payments-settings

Daily Cron: ✔ Next scheduled: 2025-07-18 19:10:53 -05:00
Options: ✔
Notes: 102
Onboarding: completed

### Braintree (Credit Card) ###

Environment: Sandbox
Tokenization Enabled: ✔
Debug Mode: Save to Log

### Braintree (PayPal) ###

Environment: Sandbox
Tokenization Enabled: ✔
Debug Mode: Save to Log

### Action Scheduler ###

Complete: 68
Oldest: 2025-07-13 19:08:52 -0500
Newest: 2025-07-18 12:54:50 -0500

Failed: 186
Oldest: 2024-09-06 00:28:33 -0500
Newest: 2025-07-16 09:12:13 -0500

Pending: 6
Oldest: 2025-07-18 21:45:22 -0500
Newest: 2025-09-08 14:49:23 -0500


### Status report information ###

Generated at: 2025-07-18 13:27:25 -05:00
```
</details>

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by amberhinds** [comment]
cc @jamesckemp @vikrampm1 @jeffpaul

**Comment by peterwilsoncc** [comment]
The styles for the pre-orders product page uses those included in WooCommerce Core, so this will need to be resolved with a PR against this repository. The same issue is apparent for the "Add to cart" button on my development install without WooCommerce Pre-Orders enabled.

<img width="520" height="804" alt="Image" src="https://github.com/user-attachments/assets/abb88027-f93a-426b-928d-c097f97dc279" />
